code/modules/mob/living/living.dm

```diff
@@ -438,6 +438,11 @@
 	if (s_active && !(s_active.ClickAccessible(src, depth=STORAGE_VIEW_DEPTH) || s_active.Adjacent(src)))
 		s_active.close(src)
 
+	for(var/mob/M in view(src))
```

Comment:
is ```src``` in their own ```view()```? make sure they're not, otherwise you're running ```update_vision_cone()``` on them twice, and we need as much speed with this as we can get.